app/Livewire/GlobalSearch.php

```
@@ -0,0 +1,371 @@
+                return str_contains($item['search_text'], $query);
+            })
+            ->take(20)
+            ->values()
+            ->toArray();
+    }
+
+    public function render()
+    {
+        return view('livewire.global-search');
+    }
+}
```

PHPMD (2.15.0)

The class GlobalSearch has a coupling between objects value of 13. Consider to reduce the number of dependencies under 13. (undefined)

(CouplingBetweenObjects)